Restructuring Commerce events (#244)
AdobeDocs/commerce-services: src/pages/shared-services/storefront-events/sdk/install.md

---
title: Install the Storefront Events SDK | Commerce Services 
description: Learn how to install the storefront events SDK.
keywords:
  - Events
  - Services
  - Storefront
---
# Install the Storefront Events SDK

The storefront events SDK serves as the foundation for eventing on an Adobe Commerce storefront. It provides access to a common data layer, and an event publishing and subscription service. You can use the [Adobe Commerce Event Collector](../collector/index.md) package to listen for the events and send them to Adobe Commerce for processing.

This SDK can be used as a hosted script, or bundled in a JavaScript application. The script version is hosted on [cdn](https://cdn.jsdelivr.net/npm/@adobe/magento-storefront-event-collector@1/dist/index.js), and the bundled version is hosted on [npm](https://www.npmjs.com/package/@adobe/magento-storefront-events-sdk).

<InlineAlert variant="info" slots="text"/>

If you are using the Data Connection module, you can skip this installation topic as the module [automatically deploys the SDK for you](https://experienceleague.adobe.com/en/docs/commerce/data-connection/fundamentals/connect-data#data-collection).

> 
> To load the SDK as a script, use the following snippet.
> 
> ```html
> <script src="https://cdn.jsdelivr.net/npm/@adobe/magento-storefront-events-sdk@1/dist/index.js"></script>
> ```
> 
> To install the script as a dependency, run this command.
> 
> ```bash
> npm install @adobe/magento-storefront-events-sdk
> ```

Reviewer: ```suggestion

- **Use the SDK as a hosted script**—Use the following snippet to load the script from the [JSDelivr CDN](https://www.jsdelivr.com/).

  ```html
  <script src="https://cdn.jsdelivr.net/npm/@adobe/magento-storefront-events-sdk@1/dist/index.js"></script>
  ```

- **Use the bundled version**—Run the following command to install the script as a dependency.

  ```bash
  npm install @adobe/magento-storefront-events-sdk
  ```

```
